scripts/overton/duplicate_filter_script.py

```diff
@@ -0,0 +1,78 @@
+import pandas as pd
+import typer
+
+def main(overton_excel_path : str, duplicate_only_path : str, no_duplicates_path : str, duplicate_report_path : str):
+
+    data = pd.read_excel(overton_excel_path)
+    df   = pd.DataFrame(data)

```

Comment:
tej linijki trochę chyba nie rozumiem, data to już powinien być df

Reply:
A faktycznie